genesis/src/lib.rs

```diff
@@ -275,7 +430,7 @@ impl<S> RawGenesisBlockBuilder<S> {
 
     /// Add instruction to the end of genesis transaction
     pub fn append_instruction(mut self, instruction: impl Into<InstructionBox>) -> Self {
```

Comment:
I guess it should be renamed too:

```suggestion
    pub fn push_instruction(mut self, instruction: impl Into<InstructionBox>) -> Self {
```

Or better keep using `append_instruction` everywhere - this will reduce code changes!